probe-rs/src/debug/source_instructions.rs

```diff
@@ -288,22 +295,23 @@ impl SourceLocation {
     ) -> Option<SourceLocation> {
         debug_info
             .find_file_and_directory(&program_unit.unit, instruction_location.file_index)
-            .map(|(file, directory)| SourceLocation {
+            .map(|path| SourceLocation {
                 line: instruction_location.line.map(std::num::NonZeroU64::get),
                 column: Some(instruction_location.column),
-                file,
-                directory,
+                path,
             })
     }
 
+    /// Get the file name of the source file
+    pub fn file_name(&self) -> Option<String> {
+        self.path
+            .file_name()
+            .map(|name| String::from_utf8_lossy(name).to_string())
+    }
+
     /// Get the full path of the source file
     pub fn combined_typed_path(&self) -> Option<TypedPathBuf> {
```

Comment:
Don't we want to remove this method? :) Or at least rename it? :)

Reply:
I've removed it.